src/sys_uv.zig

```diff
@@ -37,7 +37,23 @@ pub const mkdirOSPath = bun.sys.mkdirOSPath;
 
 // Note: `req = undefined; req.deinit()` has a saftey-check in a debug build
 
-pub fn open(file_path: [:0]const u8, flags: bun.Mode, perm: bun.Mode) Maybe(bun.FileDescriptor) {
+// convert linux flags to windows flags
+pub fn normalizeOpenFlags(flags: bun.Mode) c_int {
+    if (comptime Environment.isWindows) {
+        var win_flags: c_int = 0;
+        if (flags & std.os.O.RDONLY != 0) win_flags |= 0o0;
```

Comment:
can you document where these flags are from or use `std.os.windows.`* flags.

Reply:
I have this same code in another PR lol

Reply:
@Jarred-Sumner do you want to merge your PR then and close this one? What do you think?